### PR TITLE
Add support for indirect table in 'table get' command

### PR DESCRIPTION
**Pull request by tatry**

### CODE REVIEW

File: lib/psabpf_table.c [hunk 1]
```diff
@@ -105,14 +113,21 @@ static bool member_is_direct_object(psabpf_btf_t *btf, const struct btf_member *
...
-    if (!btf_is_struct(type))
-        return false;
     *type_name = btf__name_by_offset(btf->btf, type->name_off);
     if (*type_name == NULL)
         return false;
     if (strcmp(*type_name, "bpf_spin_lock") == 0)
         return false;
 
+    if (btf_is_int(type)) {
+        /* threat ActionSelector and ActionProfile references as a direct object */
```
Comment (kmateuszssak) [inline lib/psabpf_table.c:123]: But `ActionSelector` and `ActionProfile` are not direct objects but table implementation attributes. I suggest to create separate method like `mebmer_is_implementation_attribute` or similar.

Comment (kmateuszssak) [reply lib/psabpf_table.c:123]: Or rename `member_is_direct_object` to `..is_direct_or_implementation_object`

Comment (tatry) [reply lib/psabpf_table.c:123]: Renamed

Comment (osinstom) [inline lib/psabpf_table.c:123]: treat?

**Pull request merged**